### PR TITLE
Priority fix: SingleSpaceAfterConstructFixer must run before BracesFixer

### PR DESCRIPTION
**Pull request by kubawerlos**
Fixes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5559

### CODE REVIEW

**Comment by kbond** [comment]
After upgrading to 2.18.4, `function_declaration` (space `one`) is no longer working - will this fix that as well?

**Comment by kubawerlos** [comment]
@kbond I'm afraid no, can you report an issue with some sample code?

**Review by keradus — APPROVED** [review]

File: tests/Fixtures/Integration/priority/single_space_after_construct,braces.test [hunk 1]
```diff
@@ -0,0 +1,19 @@
+--TEST--
+Integration of fixers: single_space_after_construct,braces.
```
Comment (keradus) [inline tests/Fixtures/Integration/priority/single_space_after_construct,braces.test:2]: (yes, I know I don't like the order here. let's discuss in other thread to not block this PR)

**Comment by keradus** [comment]
Thank you @kubawerlos.

File: tests/Fixtures/Integration/priority/single_space_after_construct,braces.test (hunk 1 above)
Comment (keradus) [inline tests/Fixtures/Integration/priority/single_space_after_construct,braces.test:2]: (yes, I know we have open discussion here. let's discuss in other thread to not block this PR)

**Pull request merged**